quodlibet/ext/query/missing.py

```diff
@@ -17,7 +17,7 @@ class MissingQuery(QueryPlugin, PluginConfigMixin):
     PLUGIN_NAME = _("Missing Query")
     PLUGIN_DESC = _("Matches songs without the given tag.")
     key = 'missing'
-    usage = "<b><tt>@(missing: artist)</tt></b>"
+    usage = "<b><tt>" + _("@(missing: artist)") + "</tt></b>"
```

Comment:
Is this translatable? `missing` is fixed (name of the query plugin) and `artist` is fixed (tag name)...

Reply:
Oh, yes, totally missed that.